src/GovUk.Education.ExploreEducationStatistics.Content.Services/MethodologyService.cs

```diff
@@ -114,6 +114,7 @@ private async Task<List<PublicationSummaryViewModel>> GetPublishedPublicationsFo
                     Id = pm.PublicationId,
                     Title = pm.Publication.Title,
                     Slug = pm.Publication.Slug,
+                    ReleaseSlug = pm.Publication.LatestPublishedReleaseVersion!.Release.Slug,
```

Comment:
I think there's some unit tests in `Content.Services.Tests.MethodologyServiceTests` for the method `GetLatestMethodologyBySlug` which make assertions on the view model including the publications. These might need updating to check that `ReleaseSlug` has the right value.